server/src/main/java/org/opensearch/indices/NodeIndicesStats.java

```diff
@@ -284,4 +367,28 @@ public List<IndexShardStats> getShardStats(Index index) {
     static final class Fields {
         static final String INDICES = "indices";
     }
+
+    /**
+     * Levels for the NodeIndicesStats
+     */
+    public enum levels {
```

Comment:
Please follow the right language naming conventions.

Reply:
Sure Vikas, will fix these before raising the PR. 
Thanks